bin/pocs_shell

```diff
@@ -15,6 +15,7 @@ from astropy.io import fits
 from astropy.utils import console
 
 from pocs import POCS
+from pocs.observatory import Observatory
```

Comment:
As I remarked in another PR, this was not something that we explicitly decided on, we only talked about using root relative syntax. Either style is correct and the thing that is usually stressed is to be consistent. Since 95% of the current code uses the `from` syntax I'm not sure what the arguments are for in favor of changing them all. It is slightly more obvious what package is being referred to so I could understand wanting to switch to it, but that is a conversation we have not had yet.

Reply:
I think you made a comment but disappeared. I looked at the google style code a bit more and see what they are getting at. It's not explicitly saying not to use the `from` style but to use just the module without prefix, which is not what I am doing. Will fix accordingly.